scripts/build.py

```diff
@@ -1349,22 +1350,22 @@ def compile_typescript_files_continuously(project_dir):
                     return
 
 
-def build():
+def main(argv=None):
     """The main method of this script.
 
     Creates a third-party directory where all the JS and CSS dependencies are
     built and stored. Depending on the options passed to the script, might also
     minify third-party libraries and/or generate a build directory.
     """
-    parser = optparse.OptionParser()
-    parser.add_option(
+    parser = argparse.ArgumentParser()
```

Comment:
Please stick to your convention of defining this as a top-level constant.

Reply:
Done